packages/edit-site/src/components/global-styles/font-library-modal/fonts-grid.js

```diff
@@ -42,9 +42,13 @@ function FontsGrid( { title, children, pageSize = 32 } ) {
 				<div className="font-library-modal__fonts-grid__main">
 					{ items.map( ( child, i ) => {
 						if ( i === itemsLimit - 1 ) {
-							return <div ref={ setLastItem }>{ child }</div>;
+							return (
+								<div key={ i } ref={ setLastItem }>
```

Comment:
It is [not recommended](https://react.dev/learn/rendering-lists#why-does-react-need-keys) to use the index as its key when rendering the list. Doesn't the child object have some kind of unique key instead?

Reply:
Oh of course 🤦‍♀️ And there's a `key` on the child object! Updated here: https://github.com/WordPress/gutenberg/pull/57939/commits/2d3abce670f7ebdfb7650160427fa24775fbaf69. Thank you!